### PR TITLE
Add scalaris coins.

### PR DESCRIPTION
**Pull request by mraksoll4**
i go to folder

`C:\Program Files\AtomicDEX Desktop\assets\tools\mm2 `

then edite file 

`coins `

add this 

	{
		"coin": "SCA",
		"name": "scalaris",
		"fname": "Scalaris",
		"rpcport": 42510,
		"txversion": 1,
		"pubtype": 63,
		"p2shtype": 23,
		"wiftype": 154,
		"txfee": 20000,
		"segwit": false,
		"required_confirmations": 3,
		"avg_blocktime": 1,
		"mm2": 1,
		"protocol": {
			"type": "UTXO"
		}
	},

wwww

then go to folder 

`C:\Program Files\AtomicDEX Desktop\assets\config`

edit 

`0.4.0-coins.json`

Then i was go after start and close app to 

`C:\Users\Username\AppData\Roaming\atomic_qt`

delete at 

`C:\Users\V\AppData\Roaming\atomic_qt\0.4.0\configs`

`coins.json`

and at

`C:\Users\Username\AppData\Roaming\atomic_qt\config`

`0.4.0-coins.Test.json`

next

i run again all and it's recreate my custom files.

i add custom coin  ( Scalaris )

 
 
then send some coins to dex

`
https://explorer.scalaris.info/tx/1bab7183306c57633bbfe0e9f7e8a312755eb89b51a207ecee1acd78f852274e
`

Create  placer order

then take if from another copy dex  and wait.

[Taker fee tx hash](https://kmdexplorer.io/tx/2318139968b7a9b3a539da26456c83fa12847d8e7142c83f823125ca191caae5)

 [Got maker payment](https://explorer.scalaris.info/tx/2721d59d67b4fd32aa53c146368c6e9019d4ba6ebae99777d6828d7f3aa9ea0c)

 [Taker payment tx hash](https://kmdexplorer.io/tx/938836692a1c7b5cf65ae69345eb94901ab53b780141eebc6313677151580800)
[Taker payment spend tx](https://kmdexplorer.io/tx/dd0206817967988c2ab92e3a253f27ce0b2b7a00e9d8dc747215f09751e29368)
[Maker payment spend tx ](https://explorer.scalaris.info/tx/3a792ee41f4298d4c3d33ccb67fa7a5b532609d48ce2d2eff33d3cf56de4873f)

User Latest release .

### CODE REVIEW

**Comment by mraksoll4** [comment]
About problems with swaps , "coin" file for mm2 is not parsed cheek theme so estimate fee not work

**Comment by Milerius** [comment]
@cipig could you review this one ? (try a swap maybe)

File: assets/config/0.4.0-coins.json [hunk 1]
```diff
@@ -1493,6 +1493,29 @@
...
+        "url": "electrum2.scalaris.info:10001"
+      },
+      {
+        "url": "electrum3.scalaris.info:10001"
+      }
+    ],
+    "explorer_url": [
+      "https://explorer.scalaris.info/"
+    ],
+    "active": true,
```
Comment (cipig) [inline assets/config/0.4.0-coins.json:1516]: should be `false`

File: assets/config/0.4.0-coins.json [hunk 2]
```diff
@@ -1493,6 +1493,29 @@
     "active": false,
     "currently_enabled": false
   },
+  "SCA": {
+    "coin": "SCA",
+    "name": "Scalaris",
+    "type": "UTXO",
+    "coingecko_id": "scalaris",
+    "coinpaprika_id": "sca-scalaris",
```
Comment (cipig) [inline assets/config/0.4.0-coins.json:1501]: can't find this ids, if SCA is not listed on gecko/paprika, this should be set to `test-coin`

**Comment by cipig** [comment]
There are no orders up. @mraksoll4 can you send me some SCA to SRA7sLheTHe57GS1qHAb5JXEJuhG1EcMA3 ?

**Comment by mraksoll4** [comment]
> There are no orders up. @mraksoll4 can you send me some SCA to SRA7sLheTHe57GS1qHAb5JXEJuhG1EcMA3 ?

https://explorer.scalaris.info/tx/0e941526ebd880c271a183b96000be32648dda5e414f4d595ceecfd91122a949

**Review by cipig — APPROVED** [review]
image
swap works fine

**Review by Milerius — APPROVED** [review]

**Pull request merged**